compiler/util/files.cpp

```diff
@@ -424,13 +425,48 @@ const char* createDebuggerFile(const char* debugger, int argc, char* argv[]) {
   return dbgfilename;
 }
 
-const std::string runUtilScript(const char* script) {
+std::string runPrintChplEnv(std::map<std::string, const char*> varMap) {
+  // Run printchplenv script, passing currently known CHPL_vars as well
+
+  std::string command = "";
+  std::string result = "";
+  char buffer[256];
+
+  // Pass known variables into printchplenv via command
+  for( std::map<std::string, const char*>::iterator ii=varMap.begin(); ii!=varMap.end(); ++ii)
+  {
+    command += ii->first + "=" + std::string(ii->second) + " ";
+  }
+
+  command += std::string(CHPL_HOME) + "/util/printchplenv --simple";
+
```

Comment:
Yea. `runUtilScript` prepends the command with `CHPL_HOME + "/util/"`, but we need our final command to be something like `CHPL_FOO=1 CHPL_BAR=2 ... CHPL_HOME/util/printchplenv --simple`.